internal/pkg/dependencies/project.go

```diff
@@ -44,7 +45,7 @@ func NewProjectDeps(ctx context.Context, base Base, public Public, tokenStr stri
 func newProjectDeps(base Base, public Public, token storageapi.Token) (*project, error) {
 	// Require master token
 	if !token.IsMaster {
-		return nil, errors.New("a master token of a project administrator is required")
+		return nil, MasterTokenRequiredError{}
```

Comment:
Nedavno sme tu pridavali tuto chybu, aby uzivatel CLI/API hned vedel, ze pouziva zly token, nie az ked spadne nejaky call na Storage API.

No `errors.New` je obycajna genericka chyba, a nema nastaveny ziadny HTTP code, takze v Templates API to skonci ako `500 - Internal Error`, ale ma to byt `401 - Unauthorized` (a nema sa to logovat ako chyba).